packages/components/psammead-timestamp/src/index.jsx

```diff
@@ -34,6 +34,7 @@ const Timestamp = ({
   padding,
   service,
   darkMode,
+  className,
 }) => (
   <StyledTimestamp
```

Comment:
This component seems to be unnecessary - it's just drilling props untouched

I think deleting line 29-49 and just exporting the `StyledTimestamp` directly might be better

Reply:
> untouched

Not entirely - it's renaming `datetime` to `dateTime`